src/lib/components/obs-list/ObsItem.js

```diff
@@ -359,6 +358,7 @@ export function CategoricalObs({
         max={max}
         onChange={toggleObs}
         showColor={showColor}
+        estimateSize={42}
```

Comment:
I was still having the problem. 

I think the issue here is that the virtualized list doesn't account for any margin, which is causing the discrepancy in height. In the first instance, when calculating the total height it seems to add up all the internal heights, which is only 38. I've changed things so that each list items is now wrapped inside a containing div that has padding (this replicates the margin effect).

The height is now working as expected or me. Could you please check this, and let me know?